docs/release-notes/index.md

```diff
@@ -66,6 +66,46 @@ or anything else, please submit a [GitHub issue][issues].
   <a slot="headline" href="https://github.com/RedHat-UX/red-hat-design-system/releases">Changelog</a>
 </rh-tile>
 
+<section aria-labelledby="version-2.1.0">
+
+## Version 2.1.0
+Released October 1, 2024
+
+### Highlights
+
+<rh-table>
+
+| Change                              | Type    | Notes |
+| ----------------------------------- | ------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Theming: added theming tokens to most elements | {{i()}} | New theming tokens allow page-level and container-level colour palettes on patterns and elements. |
```

Comment:
This is the only entry that ends with a period--in v2.1 anyway. But all of the <=v2.0 entries end in periods.

Reply:
This is why I asked you to review 😅